infra/grafana/config/grafana.ini

```diff
@@ -10,3 +10,6 @@ enabled = true
 
 [users]
 default_theme = dark
+
+[feature_toggles]
+enable = tempoServiceGraph tempoSearch tempoBackendSearch traceToMetrics traceqlEditor
```

Comment:
tempoSearch and tempoBackendSearch will be on by default pretty soon (9.1.x or 9.2 of Grafana) but still need them for a little bit.